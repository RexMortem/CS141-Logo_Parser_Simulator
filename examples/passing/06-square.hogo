; This program prints a square by using the "repeat" command.
repeat 4 [
  forward 100
  right 90
]