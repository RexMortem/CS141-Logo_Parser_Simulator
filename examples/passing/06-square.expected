[Repeat 4 [GoForward 100, TurnRight 90]]